clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/utils/IsolatedIngestionUtils.java

```diff
@@ -308,7 +308,11 @@ public static void releaseTargetPortBinding(int port) {
    */
   public static Optional<Integer> getLingeringIngestionProcessId(int port) {
     Optional<Integer> isolatedIngestionProcessPid = Optional.empty();
-    String processIds = executeShellCommand("/usr/sbin/lsof -t -i :" + port);
```

Comment:
Why do we need the `/usr/sbin` prefix? It should be in the path by default

Reply:
A full path is required for CentOS/RHEL. The path is not the same on Ubuntu and it works without the path on it.  

Also, GH actions runner doesn't have CentOS and hence I'm using Ubuntu.